.github/ISSUE_TEMPLATE/create_issue.yml

```diff
@@ -0,0 +1,59 @@
+name: Found a bug?
+description: Report your bug here.
+labels: ["bug", "triage"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        #### Before you get started
+        * Check that sure someone hasn't already opened a similar [issue](https://github.com/weaviate/weaviate-io/issues).
```

Comment:
Maybe change `Check that sure someone [...]` to `Check to make sure that someone [...]`

Reply:
done